volto/i18n.rst

```diff
@@ -0,0 +1,242 @@
+.. _i18n-label:
+
+====================
+Internationalization
+====================
+
+Making Text Translatable
+========================
+
+Volto uses :file:`react-intl` to translate strings. We start by importing the
+component:
+
+::
```

Comment:
Always use an explicit lexer in a `.. code-block::` directive. Never rely on the default lexer of `::`, which is usually Python and therefore wrong.